mqtt/src/main/scala/Mqtt.scala

```diff
@@ -11,6 +11,19 @@ import org.eclipse.paho.client.mqttv3.{ MqttMessage => PahoMqttMessage, _ }
 import scala.util._
 import scala.language.implicitConversions
 
+/**
+ * Quality of Service constants as defined in
+ * https://www.eclipse.org/paho/files/mqttdoc/Cclient/qos.html
+ */
+object QoS {
+  def atMostOnce = 0
+  def atLeastOnce = 1
+  def exactlyOnce = 2
+}
```

Comment:
Better. But I would have liked a typed encoding even better. As in sealed trait + three objects (with a byte value method each for the internal representation).